dev/com.ibm.ws.concurrent.mp.1.2_fat_tck/fat/src/com/ibm/ws/concurrent/mp/v1_2/fat/tck/MPContextPropagationTCKLauncher.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018,2021 IBM Corporation and others.
+ * Copyright (c) 2018,2021,2022 IBM Corporation and others.
```

Comment:
max two years lists - first, last
```suggestion
 * Copyright (c) 2018 ,2022 IBM Corporation and others.
```

Reply:
same comment for the other files!